CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ command line for details.
 
 ### [4.4.0] - 2021-05-26
 
+#### Breaking change
+
+By mistake we released this as version 4.4.0 instead of 5.0.0 even we introduced
+a breaking change by dropping support for using
+[statsd](https://github.com/statsd/statsd#readme) metrics.
```

Comment:
```suggestion
- By mistake we released this as version 4.4.0 instead of 5.0.0 even though we introduced
  a breaking change in [#314](https://github.com/jupyterhub/configurable-http-proxy/pull/314) by
  dropping support for using [statsd](https://github.com/statsd/statsd#readme) metrics.
```

nit but some formatting and context on the breaking change